src/components/input.js

```diff
@@ -174,12 +174,10 @@ export const NumberInput = ({ onChange, onBlur, min = -Infinity, max = Infinity,
  * @param {object} props.inputProps
  * @param {object} [props.error] - error message content
  */
-export const ValidatedInput = props => {
-  const { inputProps, error } = props
-
+export const ValidatedInput = ({ inputProps, error }) => {
   return h(Fragment, [
     div({
-      style: { position: 'relative', display: 'flex', alignItems: 'center' }
+      style: { position: 'relative', display: 'flex', alignItems: 'center', width: inputProps?.style?.width }
```

Comment:
TOL: This works when setting a `width`, but wouldn't work when using other styles that affect the size, e.g. `maxWidth`. Another way to handle this would be to have the consumer create a width-constrained wrapper around the input. That should work because our inputs take up 100% of the available space by default.

Reply:
That occurred to me, but I also felt that `ValidatedInput` should render consistently and correctly regardless of container. Another thought: instead of reusing `inputProps.style.width`, we could add a `width` prop to `ValidatedInput` that gets applied to the wrapping container already exists in `ValidatedInput`. Thoughts?

Reply:
That seems a little better and more explicit than hooking onto the style, yes.

Reply:
I think I added this feedback correctly, would one of you mind checking?